### ISSUE TITLE
openstack node: trinity_api problematic on reboot

### ISSUE THREAD

**Issue by themisa** [clustervision/trinity#313]
When we reboot the openstack node, trinity_api is not running

(service trinity status reports not running, systemctl status trinity_api.service reports running, but with issues)

No /var/run/trinity

**Comment by themisa** [comment]
```
[root@openstack ~]# service trinity_api status
Not Running.
[root@openstack ~]# systemctl status trinity_api.service
trinity_api.service - LSB: Trinity API server
   Loaded: loaded (/etc/rc.d/init.d/trinity_api)
   Active: active (running) since Wed 2015-06-24 13:17:35 CEST; 1min 39s ago
  Process: 2651 ExecStart=/etc/rc.d/init.d/trinity_api start (code=exited, status=0/SUCCESS)
   CGroup: /system.slice/trinity_api.service
           3223 /usr/bin/python /usr/bin/trinity_api

Jun 24 13:17:33 openstack.cluster systemd[1]: Starting LSB: Trinity API server...
Jun 24 13:17:35 openstack.cluster trinity_api[2651]: Starting trinity_api:/etc/rc.d/init.d/trinity_api: line 40: /var/run/trinity/trinity_api.pid: No such file or directory
Jun 24 13:17:35 openstack.cluster trinity_api[2651]: [  OK  ]
Jun 24 13:17:35 openstack.cluster systemd[1]: Started LSB: Trinity API server.
```

**Comment by abhimukh** [comment]
This should now be fixed in r5 (6b09cc6f0fd0c94bf8b26803015c6bc1c580d6ae). 

For earlier branches, manual workaround

```
# systemctl stop trinity_api.service
# mkdir -p /var/run/trinity
# systemctl start trinity_api.service 
```

**Comment by themisa** [comment]
Fixed in * r5
2d04330


[root@openstack ~]# service trinity_api status
2217
Running.
